tutorials/proto_rename/person.pb

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
We don't need this. We'll generate it as part of the tutorial.
Let's have `read` and `write` commands.
So you would run, e.g:
`bazel run //... -- read`

Reply:
Deleted this file.